mep/books/views.py

```diff
@@ -17,6 +19,15 @@ class ItemList(LabeledPagesMixin, ListView):
     def get_queryset(self):
         return ItemSolrQuerySet().order_by('title')
 
+    def get_uri(self):
+        return absolutize_url(reverse('books:books-list'))
+
+    def get_breadcrumbs(self):
```

Comment:
Worth noting somewhere that this can't be set on the view `breadcrumbs` attribute because the url conf isn't loaded in time? 

Also, if that's the case will we ever be able to use the breadcrumbs attribute?  Or is there a view hook we could use to set it once (init, maybe?) instead of every time?

Reply:
Not sure, seems like something to revisit though. I'll add a note

Reply:
Would be nice if we could at least use it for auto-adding the home crumb, which doesn't require the urlconf and is at the start of all sets of crumbs.